terms/schema.md

```diff
@@ -10,6 +10,7 @@ By "defined semantics" we mean that it is expressed in a particular [schema lang
 Schema allow you to:
 
 * de-risk projects involving data,
+* Scope a data campaign. This is how to be strategic about what data you capture
```

Comment:
There's no corresponding example for this, so we can either add one or not include this.

Probably the best thing to do to keep things succinct is to modify the prior line to "scope and de-risk projects involving data"

Reply:
Perhaps there should be an example about scoping a data campaign. Something like this:

It is important for all parties involved in data generation and transformation to have transparent expectations around what data to collect. Schemas allow for boxes to be checked when collecting data. For example, it would be very useful for an anemometer's orientation to be recorded by the installation technician, in a way that this metadata is accessible to data analysis.